release-please-config.json

```diff
@@ -1,8 +1,20 @@
 {
-  "bootstrap-sha": "2b2880476eb3b4922330b3f907ad01735a03cc3e",
+  "bootstrap-sha": "97925df8f4721bb222e9e01e73b39c9957572255",
   "packages": {
     "db-service": {},
     "sqlite": {},
     "postgres": {}
-  }
+  },
+  "changelog-sections": [
+    {
+      "type": "feat",
+      "section": "Features",
+      "hidden": false
+    },
+    {
+      "type": "fix",
+      "section": "Fixed",
+      "hidden": false
```

Comment:
also added mappinggs from the conventional commit type to the respective section in the CHANGELOG.
We will not need the section `Changed` anymore.